parsec/backend/invite.py

```diff
@@ -152,6 +156,488 @@ def _on_status_changed(event, organization_id, greeter, token, status):
 
         self._event_bus.connect("invite.status_changed", _on_status_changed)
 
+    def send_invite_email(self, invitation):
+        # Temporary infos, idk from which address the mail is going to be sent
+        # (and I don't know how to secure the authentification, which is why
+        #   I'm using a test email address with password directly written here)
+        # receiver_email can be anything, will later be changed to be the
+        #   invited's email
+        sender_email = "pytestmailtest@gmail.com"
+        receiver_email = "pytestmailtest@gmail.com"
+        password = "mailtest"
+        
+        # mail settings
+        message = MIMEMultipart("alternative")
+        message["Subject"] = "You have received an invitation on Parsec Cloud"
+        message["From"] = sender_email
+        message["To"] = receiver_email
+        
+        # Since I'm using the text twice, I've made these strings to store it
```

Comment:
Small nitpickings:
- we don't use first person in the comments (I guess it's because code is shared among devs with no sens of ownership ^^)
- I don't think the tabulation for multiple lines sentence is useful (looks like it decreases readability). The only place I personally do this is when having a bullet point list with some line with multiple lines.

And good job at doing this kind of comments, giving context to code is good ;-)